metadata.mk

```diff
@@ -22,6 +22,8 @@ UBI_VERSION=8.10
 # Configuration for Semaphore integration.
 ORGANIZATION = projectcalico
 
+REPO_ROOT := $(shell git rev-parse --show-toplevel)
```

Comment:
metadata.mk is intended to be statically configured metadata that varies per-branch / per-product / etc. 

There's already a REPO_ROOT in `lib.Makefile` that we should use instead, and remove this from here.